scipy/stats/morestats.py

```diff
@@ -2706,17 +2706,27 @@ def circmean(samples, high=2*pi, low=0, axis=None):
     Returns
     -------
     circmean : float
+        nan value if no mean can be calculated (e.g., uniform 
+            circular distribution)
```

Comment:
I suspect it's this indent which throws refguide-check off.  You can try `$ python runtests.py --refguide-check -s stats` locally.